lib/series_seasonal_adjustment.rb

```diff
@@ -5,10 +5,10 @@ def apply_seasonal_adjustment(factor_application)
     ns_series = Series.get ns_series_name
     raise SeasonalAdjustmentException.new if ns_series.nil?
     set_factors factor_application 
-    new_ns_values = ns_series.get_values_after last_demetra_datestring
+    new_ns_values = ns_series.get_values_after (Date.parse last_demetra_datestring)
     adjusted_data = {}
     new_ns_values.each do |datestring, value|
```

Comment:
We could rename 'datestring' to 'date' to reflect it is not supposed to be a String.